### PR TITLE
Anchore Engine v0.10.0 devolopment branch.

### PR DESCRIPTION
**Pull request by Toure**
This is the devolopment branch which release canidates will
be base upon.

Signed-off-by: Toure Dunnon <toure.dunnon@anchore.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>)(, fixes #<issue_number, ...)` format, will close the issue when PR is merged*: fixes #:

**Special notes**:

### CODE REVIEW

**Review by nightfurys — APPROVED** [review]
lgtm if this covers all the places the version needs to be updated. Can't keep the list in my head any more, hopefully it's doc-ed somewhere

**Comment by Toure** [comment]
@nightfurys, yeah I followed the release docs and also searched the project to make sure I didn't miss anything.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 0.10.0
```
Comment (zhill) [inline CHANGELOG.md:3]: This is a good start, but we're going to need a lot more here before we release. There are deprecation notices, compatibility notes, and other things we need to highlight here.

Also should include updates to the docs/ dir and a new release notes page.

**Comment by nightfurys** [comment]
v0.10.0 will be merged back into master post release. PR was automatically closed as I deleted the branch and redid the setup. @Toure's commits are not handled #1091

**Pull request closed without merging**